site/content/en/docs/components/core/hook.md

```diff
@@ -1,8 +1,41 @@
 ---
 title: "Hook"
-weight: 30
-description: >
-  
+weight: 60
+description: PROW (Prow Hook) is a webhook service for Kubernetes that listens for events from Github, Gitlab, and other sources, and then triggers actions in response to those events. It is commonly used to automate tasks such as building and deploying code, running tests, and managing pull requests.
```

Comment:
Pls don't capitalize Prow (even below). IMO it's not even necessary to talk about "Prow Hook" in the  context of a Prow documentation page. It's just 'Hook'. 

Also, I don't think Prow can handle GitLab events (but it can does handle gerrit).

Reply:
thanks @petr-muller , changed as per comments.